pkgs/tools/system/loadwatch/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchurl, ... }:
+
+stdenv.mkDerivation {
+  name = "loadwatch-1.1";
+  src = fetchurl {
+    url = "mirror://debian/pool/main/l/loadwatch/loadwatch_1.0+1.1alpha1.orig.tar.gz";
```

Comment:
Is there no upstream url for loadwatch?

Reply:
"you can get loadwatch from ftp://rplay.doit.org/pub/loadwatch/" which is gone.

Reply:
not surprising, since this package is from 1998-2000 according to the sources.
Do we really want to include an 18-20 yrs old unmaintained package?

Reply:
Thanks @xeji for having a look. At least it's still compiling, functional and the codebase is rather small. It would be easy to review. I'll try to contact the former authors. Maybe it can be revived.